.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "bin/data/vuforia-spatial-toolbox-userinterface"]
+	path = bin/data/vuforia-spatial-toolbox-userinterface
+	url = git@github.com:ptcrealitylab/vuforia-spatial-toolbox-userinterface.git
```

Comment:
If someone doesn't have ssh setup they won't be able to use this url so https://github.com/ptcrealitylab/vuforia-spatial-toolbox-userinterface.git is a bit better

Reply:
ah ok! I'll update that